engine/lib/phx/src/lib.rs

```diff
@@ -11,19 +11,6 @@
 #![feature(extern_types)]
 #![feature(thread_local)]
 
-extern crate chrono;
-extern crate core;
-extern crate fmod_sys;
-extern crate freetype_sys;
-extern crate glam;
-extern crate image;
-extern crate libc;
-extern crate memoffset;
-extern crate mlua;
-extern crate printf_compat;
-extern crate rodio;
-extern crate sdl2_sys;
```

Comment:
Interesting, does the Rust compiler just implicitly link to these now whenever we import them?

Reply:
Most likely. I know that `extern crate` is not necessary now for quite some time (year or so).
The only place where we still need it is lua (mlua) wrapper because we export functions from it via `extern "C"`.